### DISCUSSION TITLE
Is it known that the kitty distributed with ubuntu 23.04 breaks previews?

### DISCUSSION THREAD

**Discussion by Msouza91** [sxyazi/yazi#636] [1. Q&A] [answered]
I was going to open an issue because previews for anything other than text files was slow and locking up yazi, but remembered that Ubuntu ships some ancient packages sometimes so I updated from 0.26.5 to 0.32.1 and all my issues went away.

If not known, maybe there should be a warning with a minimum working version for kitty.

**Comment by sxyazi** [comment]
Thanks for the report, I'll add it to the docs

**Comment by sxyazi** [comment, marked as answer]
Done by https://github.com/yazi-rs/yazi-rs.github.io/commit/caed002a1a6a7770e59da2d1e7ec3b5962439e53

**Reply by Msouza91** [reply]
It is concise, but maybe some less experienced user could assume that the latest version delivered by their package manager is the latest version of the software, having a minimum version requirement could make it clear when they go check their version that they need to update the software in another way? Just got me thinking.
